R/common.R

```diff
@@ -97,7 +97,7 @@ runJaspResults <- function(name, title, dataKey, options, stateKey, functionCall
   }
 
   if (! jaspResultsCalledFromJasp()) {
-    .numDecimals        <- 3
+	.numDecimals        <- 3
```

Comment:
```suggestion
    .numDecimals        <- 3
```
in R: 2 spaces per indent, no tabs!